test/unit/lib/plugins/aws/remove/index.test.js

```diff
@@ -1,57 +1,66 @@
 'use strict';
 
-const expect = require('chai').expect;
+const chai = require('chai');
 const sinon = require('sinon');
-const AwsProvider = require('../../../../../../lib/plugins/aws/provider');
-const AwsRemove = require('../../../../../../lib/plugins/aws/remove/index');
-const Serverless = require('../../../../../../lib/Serverless');
-
-describe('AwsRemove', () => {
-  const serverless = new Serverless();
-  const options = {
-    stage: 'dev',
-    region: 'us-east-1',
-  };
-  serverless.setProvider('aws', new AwsProvider(serverless, options));
-  const awsRemove = new AwsRemove(serverless, options);
-  awsRemove.serverless.cli = new serverless.classes.CLI();
+const runServerless = require('../../../../../utils/run-serverless');
 
-  describe('#constructor()', () => {
-    let validateStub;
-    let emptyS3BucketStub;
-    let removeStackStub;
-    let monitorStackStub;
+chai.use(require('chai-as-promised'));
+chai.use(require('sinon-chai'));
+const expect = require('chai').expect;
 
-    beforeEach(() => {
-      validateStub = sinon.stub(awsRemove, 'validate').resolves();
-      emptyS3BucketStub = sinon.stub(awsRemove, 'emptyS3Bucket').resolves();
-      removeStackStub = sinon.stub(awsRemove, 'removeStack').resolves();
-      monitorStackStub = sinon.stub(awsRemove, 'monitorStack').resolves();
+describe('AwsRemove', () => {
```

Comment:
In refactored test files I started the convention where we name the top describe after filename

(when tests fails it helps to point the test fail in which test fail happens, unfortunately Mocha doesn't expose that normally)

Reply:
great call, changed :+1: